Description: Follow Fedora's example and show "Document Viewer" in
 Unity & Gnome Shell search
Bug: https://bugzilla.gnome.org/show_bug.cgi?id=634245
Bug-Debian: http://bugs.debian.org/427576

Index: evince-3.2.0/data/evince.desktop.in.in
===================================================================
--- evince-3.2.0.orig/data/evince.desktop.in.in	2010-07-14 09:54:39.000000000 +0200
+++ evince-3.2.0/data/evince.desktop.in.in	2011-09-27 15:15:37.483486380 +0200
@@ -7,7 +7,6 @@
 Terminal=false
 Type=Application
 Icon=evince
-NoDisplay=true
 X-GNOME-DocPath=
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=evince
